build/Targets/Settings.props

```diff
@@ -10,9 +10,9 @@
     <DeveloperBuild>false</DeveloperBuild>
     <DeveloperBuild Condition="'$(OfficialBuild)' != 'true' AND '$(JenkinsBuild)' != 'true'">true</DeveloperBuild>
 
-    <ProjectDir>$(MSBuildThisFileDirectory)..\..\</ProjectDir>
-    <NuGetToolPath>$(ProjectDir)nuget.exe</NuGetToolPath>
-    <ToolsetPackagesDir>$(ProjectDir)build\ToolsetPackages\</ToolsetPackagesDir>
+    <RepoRoot>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)..\..\'))</RepoRoot>
```

Comment:
I'm pretty sure ProjectDir has some significant meaning in MSBuild. At the very least, this should probably be equivalent to `SolutionDir`, which is a more logically appropriate well known property.